src/app/modules/main/wallet_section/overview/module.nim

```diff
@@ -89,7 +89,8 @@ method filterChanged*(self: Module, addresses: seq[string], chainIds: seq[int],
       walletAccount.assetsLoading,
       walletAccount.colorId,
       walletAccount.emoji,
-      isWatchOnlyAccount=walletAccount.walletType == "watch"
+      isWatchOnlyAccount=walletAccount.walletType == "watch",
```

Comment:
can we add canSend: isWatchOnlyAccount=walletAccount.walletType == "watch",... may make it more readable